week8/homepage/styles.css

```diff
@@ -0,0 +1,141 @@
+.header {
+	animation-name: backgroundColorPalette;
```

Comment:
css sollte zwei spaces eingerückt sein. Schau mal in deinen Editor Einstellungen.